### PR TITLE
Placement strategies are now working.

### PR DESCRIPTION
**Pull request by dkirillov**
@BrunoCarrier

### CODE REVIEW

**Comment by dkirillov** [comment]
screen shot 2017-11-20 at 4 03 33 pm
screen shot 2017-11-20 at 4 03 49 pm
screen shot 2017-11-20 at 4 04 00 pm
screen shot 2017-11-20 at 4 04 20 pm

File: app/scripts/modules/ecs/placementStrategy/placementStrategy.service.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  public getPredefinedStrategy(strategyName: string): PlacementStrategy[] {
+    if ( strategyName === 'AZ Balanced Spread' ) {
+      return this.getAzBalancedSpreadStrategy();
+    } else if ( strategyName === 'AZ Balanced BinPack' ) {
+      return this.getAzBalancedBinPackStrategy();
+    } else if ( strategyName === 'BinPack' ) {
+      return this.getBinPackStrategy();
+    } else if ( strategyName === 'One Task Per Host' ) {
+      return this.getOneTaskPerHostStrategy();
+    } else {
```
Comment (BrunoCarrier) [inline app/scripts/modules/ecs/placementStrategy/placementStrategy.service.ts:16]: You could add a TODO that indicates that custom needs to be implemented as well.  but that's super optional

File: app/scripts/modules/ecs/serverGroup/configure/wizard/advancedSettings/advancedSettings.component.html [hunk 2]
```diff
@@ -23,15 +23,13 @@
       <b>Placement strategy</b> <help-field key="ecs.placementStrategy"></help-field>
     </div>
     <div class="col-md-7">
-      <ui-select multiple
-                 ng-model="$ctrl.command.placementStrategy"
-                 class="form-control input-sm">
-        <ui-select-match>{{$item}}</ui-select-match>
-        <ui-select-choices repeat="item in ['AZ Balanced Spread', 'AZ Balanced BinPack', 'BinPack', 'One Task Per Host']">
+      <ui-select ng-model="$ctrl.command.placementStrategyName" class="form-control input-sm">
```
Comment (BrunoCarrier) [inline app/scripts/modules/ecs/serverGroup/configure/wizard/advancedSettings/advancedSettings.component.html:26]: please add a help pop-up that says that custom strategies are not yet implemented and PRs are welcome

File: app/scripts/modules/ecs/placementStrategy/placementStrategy.service.ts (hunk 1 above)
Comment (dkirillov) [reply app/scripts/modules/ecs/placementStrategy/placementStrategy.service.ts:16]: Done.

File: app/scripts/modules/ecs/serverGroup/configure/wizard/advancedSettings/advancedSettings.component.html (hunk 2 above)
Comment (dkirillov) [reply app/scripts/modules/ecs/serverGroup/configure/wizard/advancedSettings/advancedSettings.component.html:26]: Done.

**Comment by BrunoCarrier** [comment]
+2 LGTM

**Pull request merged**